### PR TITLE
[refinery] Prepare v1.7.0 release

### PR DESCRIPTION
**Pull request by MikeGoldsmith**
### Description
Prepeares thes Refinery v1.7.0 chart release.

### Changes
- Adds changlog entry
- Updates chart version to 1.7.0

### CODE REVIEW

File: charts/refinery/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Refinery Helm Chart Changelog
 
+## Refinery v1.7.0
+
+- Bump app version to [v1.12.0](https://github.com/honeycombio/refinery/releases/tag/v1.12.0) (#125) | [@MikeGoldsmith](https://github.com/MikeGoldsmith)
```
Comment (vreynolds) [inline charts/refinery/CHANGELOG.md:5]: we're not super consistent about it, but in some previous chart releases we've added a short description of the underlying refinery changes corresponding to the bump

Comment (MikeGoldsmith) [reply charts/refinery/CHANGELOG.md:5]: Added a note on release changes 👍🏻

File: charts/refinery/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ## Refinery v1.7.0
 
 - Bump app version to [v1.12.0](https://github.com/honeycombio/refinery/releases/tag/v1.12.0) (#125) | [@MikeGoldsmith](https://github.com/MikeGoldsmith)
+  - this release adds prelimary support for environments and services
```
Comment (JamieDanielson) [inline charts/refinery/CHANGELOG.md:6]:
```suggestion
  - this release adds preliminary support for environments and services
```

**Review by JamieDanielson — APPROVED** [review]
nit: typo 👍

**Pull request merged**